components/layout/layout.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styles from "./layout.module.scss";
+import Link from "next/link";
+
+const Layout = ({ children }) => {
```

Comment:
Maybe we can rename this to something like NavBar remember to rename the file as well